### PR TITLE
Examples style improvments

### PR DESCRIPTION
**Pull request by rhysmdnz**

### CODE REVIEW

**Comment by dythya** [comment]
You ran something that just fixed them all?

**Comment by rhysmdnz** [comment]
Yup

File: docs/.flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F405,F403,F841,E722
```
Comment (dythya) [inline docs/.flake8:2]: ?

Comment (rhysmdnz) [reply docs/.flake8:2]: F403 and F405 are to do with import * which we aren't currently fixing.
F841 is unused variables this is mostly comes up next to commented out code that used the variable someone will need to check why this is commented out / unused in future.
E722 is using bare except statements someone needs to go look at these in the future and work out what there supposed to be catching

I've updated #50 with notes to fix these

**Review by dythya — APPROVED** [review]

File: docs/.flake8 (hunk 1 above)
Comment (dythya) [reply docs/.flake8:2]: cool

**Comment by tonyvignaux** [comment]
Congratulations, johnguant, on making these style improvements.

I observe that people are still using SimPy. This "classic" version is being used for teaching simulation at Victoria University of Wellington. Is it being used anywhere else?

**Pull request merged**